flink-runtime/src/main/scala/org/apache/flink/runtime/messages/TaskManagerMessages.scala

```diff
@@ -129,6 +129,13 @@ object TaskManagerMessages {
    * @param cause reason for the external failure
    */
   case class FailTask(executionID: ExecutionAttemptID, cause: Throwable)
+
+  /**
+   * Makes the TaskManager to disconnect from the registered JobManager
```

Comment:
Minor typo. Comment could be changed to "Disconnects the TaskManager from the JobManager"